pkgs/development/compilers/bs-platform/default.nix

```diff
@@ -1,15 +1,39 @@
-{ stdenv, fetchFromGitHub, ninja, nodejs, python3, ... }:
+{ stdenv, runCommand, fetchFromGitHub, ninja, nodejs, python3, ... }:
 let
+  build-bs-platform = import ./build-bs-platform.nix;
+
   meta = with stdenv.lib; {
     description = "A JavaScript backend for OCaml focused on smooth integration and clean generated code.";
     homepage = https://bucklescript.github.io;
     license = licenses.lgpl3;
     maintainers = with maintainers; [ turbomack gamb anmonteiro ];
     platforms = platforms.all;
+    # Currently there is an issue with aarch build in hydra
+    # https://github.com/BuckleScript/bucklescript/issues/4091
+    badPlatforms = [ "aarch64-linux" ];
```

Comment:
:)
```suggestion
    badPlatforms = platforms.aarch64;
```

Reply:
this is actually leftover :face_with_head_bandage: I moved this to https://github.com/NixOS/nixpkgs/pull/76104/files#diff-044eccb41d6037a6332d2c868d9365faR37